src/opm/parser/eclipse/Parser/Parser.cpp

```diff
@@ -792,7 +792,7 @@ void skipUDT( ParserState& parserState, const Parser& parser) {
       if (record_count < count_max)
           record_count++;
       else {
-           std::string deck_name = str::make_deck_name( line );        
+           std::string deck_name = str::make_deck_name( line );               
```

Comment:
apparently still got it wrong 😒

Reply:
git add -p is one easy way to deal with it.

Reply:
yes ok, thought it was going to be smashed anyways, but I'll rebase then 😊

Reply:
Screw-ups cleaned 🛀